src/pybind/mgr/diskprediction/predictor/models/config.json

```diff
@@ -0,0 +1,77 @@
+{
```

Comment:
my concern is the license of the pre-trained SVM models. because this dataset are pickled SVM classifiers, which by themselves are machine readable after being un-pickled. but what about its "source"? or are they in the source form already? if yes, how is user allowed to "modify" it in an effective way even he/she understands the SVM and the python language? as LGPL 2.1 requires the work to be accompanied with the source code of it. if we cannot provide the source, we will have to re-distribute these data files in a different license.

i had a hard time when preparing[0] a software package which used a statistical language model for debian. the packaged software was licensed under LGPL2.1 and CDDL. and the package was rejected by debian's FTP master because of the license of the pre-trained data: we licensed it under the same dual license.

yes, in this context, we are the upstream developers not downstream maintainer. but i think it's worthy of mention.

---
[0] https://lists.debian.org/debian-devel/2008/05/msg00005.html

Reply:
Interesting thought.  In my opinion, the original dataset would not constitute "source code" for licensing purposes, but I can see how there could be some debate.  It is probably prudent to apply a different license to the model.

Perhaps just declare the model files as public domain -- @hsiang41 , what do you think?

Reply:
Using a different license for the model files would be the easiest thing.

Reply:
@jcsp Agree. But I did not know how to do this. Do you have sample for this? or Need I apply something comment in my project?

Reply:
@hsiang41 in the top level `COPYING` file you can see a list of various exceptions.  I'd add a section at the bottom of that, and also add a COPYING file in your models/ directory that makes a statement that these particular files are donated by ProphetStor to be used by anyone for any purpose and you make no copyright claims over them.